src/openzaak/components/besluiten/openapi.yaml

```diff
@@ -2622,12 +2618,9 @@ components:
           minLength: 1
         informatieobject:
           title: Informatieobject
-          description: URL-referentie naar het INFORMATIEOBJECT (in de Documenten
-            API) waarin (een deel van) het besluit beschreven is.
+          description: URL-referentie naar het INFORMATIEOBJECT.
           type: string
           format: uri
-          maxLength: 1000
-          minLength: 1
```

Comment:
I removed length restrictions from model fields because they are not shown in the schema for `HyperlinkedRelatedField`.

Reply:
Let's discuss setting length constrants for `HyperlinkedRelatedField`:
1. I don't think that we should set max_length in the schema via Inspector without setting a real length validator, Because we will not really check if the client fits max_length requirements. 
1. As far as I get it I can't validate that the url doesn't exceed max_length, because validators are run after `to_internal_value` and the internal value of HyperlinkedRelatedField is a model instance, not a url.

Personally I'd left this urls in schema as they are until the loose-fk relations are implemented